### ISSUE TITLE
Clarify requestSizing API targetCPUUtilization parameter

### ISSUE THREAD

**Issue by souenzzo** [kubecost/docs#156]
In this docs

https://github.com/kubecost/docs/blob/3a61e569e60ec30671240efb7970a17959e4afaa/api-request-right-sizing.md

It says that `targetCPUUtilization` should be between 0 and 1. 
It does not say if this range is exclusive `(0, 1)` or inclusive  `[0, 1]` 
It actually behaves in a odd way:
- `0` returns 500: error: RecommmendRequestSizesMaxHeadroom: expected target CPU utilization to be > 0, was 0.000000
- `-1` returns a response
- `2` returns a reponse
- it is not clear which numbers notation/syntax are allowed. Is `1e-2` allowed?
- `0e` returns 200. `1abc` returns 200 too. What does it means?

I would like to know which value will be used for this parameter, if not sent

gz#1207

**Comment by kbrwn** [comment]
hi @souenzzo Thank you for this very helpful report. We will update the docs to provide further clarity on this topic soon.

**Comment by souenzzo** [comment]
An extra question: 

- do `window` support `2d3h`? does it support [ISO8601](https://en.wikipedia.org/wiki/ISO_8601#Durations) or some other date/time/duration representation specification?

**Comment by kbrwn** [comment]
@souenzzo `2d3h` is not valid. You can pass ISO8601 dates like: `window=2021-12-01T00:00:00Z,2021-12-18T00:00:00Z`

**Comment by michaelmdresser** [comment]
Hi @souenzzo, I've opened https://github.com/kubecost/docs/pull/159 to improve the documentation for this API. Let me know if this addresses your concern!

**Comment by souenzzo** [comment]
Some future concerns

> they will instead default to your savings profile's default value. If you have not changed the profile, this is `0.65`

Can `targetCPUUtilization` be returned in the response?

As it default value is configurable, it basically impossible to know the value used to a request without a `targetCPUUtilization` parameter

Also, if `targetCPUUtilization` is present in the response, will be possible to know if the `parseFloat` was successful, or if it failed silently.

(applies to every other paramters. maybe a full `parameters: {}` object in the response ?! ) 

> Duration of time over which to query. Accepts: words like ....

Would be nice if the `window` accept ISO 8601 duration spec: https://en.wikipedia.org/wiki/ISO_8601#Durations
It will allow to easily create programmatic values in other languages and easier to communicate the format.

**Comment by michaelmdresser** [comment]
I agree that this parsing behavior is really bad. I've opened https://github.com/kubecost/cost-analyzer-helm-chart/issues/1197 to track this, including your ideas.

As far as the `window` parameter goes, I appreciate this recommendation. The way we've parsed `window` has been fixed for a little while, see https://github.com/kubecost/cost-model/blob/d2ea7c577c03997a57f84b7384fc91fedf2a0197/pkg/kubecost/window.go#L117. We'd be happy to consider a pull request if you'd like to add ISO 8601 duration support.
